src/App.tsx

```diff
@@ -1,25 +1,88 @@
+import React, { useState } from 'react';
 import './App.scss';
+import classNames from 'classnames';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
 export const App = () => {
+  const [userValue, setUserValue] = useState(0);
+  const [todoTitle, setTodoTitle] = useState('');
+  const [titleError, setTitleError] = useState(false);
+  const [userError, setUserError] = useState(false);
+
+  const idCreator = () => todosFromServer.length + 1;
+
+  const createToDo = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    if (!todoTitle) {
+      setTitleError(true);
+
+      return;
+    }
+
+    if (!userValue) {
+      setUserError(true);
+
+      return;
+    }
+
+    todosFromServer.push({
+      id: idCreator(),
+      title: todoTitle,
+      completed: false,
+      userId: userValue,
+    });
+
+    setUserValue(0);
+    setTodoTitle('');
+    setTitleError(false);
+    setUserError(false);
+  };
+
   return (
     <div className="App">
       <h1>Add todo form</h1>
 
-      <form action="/api/todos" method="POST">
+      <form
+        action="/api/todos"
+        method="POST"
+        onSubmit={createToDo}
+      >
         <div className="field">
-          <input type="text" data-cy="titleInput" />
-          <span className="error">Please enter a title</span>
+          <label htmlFor="title">Title:</label>
+          <input
+            type="text"
+            id="title"
+            value={todoTitle}
+            data-cy="titleInput"
+            placeholder="Enter a title"
+            onClick={() => setTitleError(false)}
+            onChange={e => setTodoTitle(e.target.value)}
+          />
+          { titleError && <span className="error">Please enter a title</span> }
         </div>
 
         <div className="field">
-          <select data-cy="userSelect">
+          <label htmlFor="select">User:</label>
+          <select
+            data-cy="userSelect"
+            id="select"
+            value={userValue}
+            onClick={() => setUserError(false)}
```

Comment:
Avoid using onClick to clear the error. It's better to clear it onChange, as the user starts making a selection.